tests/versions/java.yml

```diff
@@ -3,24 +3,12 @@ JAVA_AGENT:
   - 'github;v1.0.1'
 
 exclude:
-  - JAVA_AGENT: 'github;v1.0.1'
-    APM_SERVER: '6.7'
-  - JAVA_AGENT: 'github;v1.0.1'
-    APM_SERVER: '6.6;--release'
-  - JAVA_AGENT: 'github;v1.0.1'
-    APM_SERVER: '6.5;--release'
   - JAVA_AGENT: 'github;v1.0.1'
     APM_SERVER: '6.4;--release'
   - JAVA_AGENT: 'github;v1.0.1'
     APM_SERVER: '6.3;--release'
   - JAVA_AGENT: 'github;v1.0.1'
     APM_SERVER: '6.2;--release'
-  - JAVA_AGENT: 'github;master'
-    APM_SERVER: '6.7'
-  - JAVA_AGENT: 'github;master'
-    APM_SERVER: '6.6;--release'
-  - JAVA_AGENT: 'github;master'
-    APM_SERVER: '6.5;--release'
   - JAVA_AGENT: 'github;master'
     APM_SERVER: '6.4;--release'
   - JAVA_AGENT: 'github;master'
```

Comment:
Same as go agent, the java agent does not support v1 with any GA version, so should be removed from this branch.